handler/commentHandler.go

```diff
@@ -208,6 +224,28 @@ func manageAssignment(req types.IssueCommentOuter, cmdType string, cmdValue stri
 	return buffer.String(), nil
 }
 
+func manageReviewers(req types.PullRequestOuter, cmdType string, cmdValue string, config config.Config) (string, error) {
```

Comment:
I am not keen on the word `manage` in `manageReviewers`. I think this might have been picked because it was used somewhere else in the Derek code (where I also mentioned this)

`manage` is not descriptive - what else could work? `editReviewers` seems more fitting